CHANGELOG.rst

```diff
@@ -1,7 +1,22 @@
 Changelog
 =========
 
+0.2.0
+`````
+
+Version 0.2.0 updates python-bsonjs's vendored copy of libbson to 1.6.5
+http://mongoc.org/libbson/1.6.2/ and fixes an error in our Linux builds.
```

Comment:
I'd be a little more specific about the build fix. You should mention that the previously published python 3.x wheels don't work.